nix/shell.nix

```diff
@@ -39,7 +39,7 @@ mkShell {
     reason
     utop
     ocamlformat
-    ocaml-lsp
+    ocaml-lsp-server
```

Comment:
I don't know if this is a variable name

Reply:
This was correct before. Nix has its own registry where it’s called ocaml-lsp

Reply:
I didn't check if in the Nix registry was called `ocaml-lsp-server` by the way. I was thinking that packages are from opam repository

Reply:
It was correct before, please revert https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/ocaml-packages.nix#L997

Reply:
👍🏻